### PR TITLE
Basemap-data tests

### PR DESCRIPTION
**Pull request by ocefpaf**
Workaround https://github.com/conda-forge/staged-recipes/pull/168#issuecomment-201380042

(We need to fix this for good.)

### CODE REVIEW

File: recipes/basemap-data-hires/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package:
+ package:
```
Comment (jakirkham) [inline recipes/basemap-data-hires/meta.yaml]: Not sure, but this might goof something.

Comment (ocefpaf) [inline recipes/basemap-data-hires/meta.yaml]: Argh. Copy-and-pasta!

Comment (jakirkham) [inline recipes/basemap-data-hires/meta.yaml]: No worries. Let's see if it goes through.

Comment (jakirkham) [inline recipes/basemap-data-hires/meta.yaml]: After all, we don't want to muck with it while it tries to remove it post-feedstocking.

Comment (jakirkham) [inline recipes/basemap-data-hires/meta.yaml]: Nevermind, we need to fix it. Linter [failed]( https://travis-ci.org/conda-forge/staged-recipes/jobs/118524074#L300 ).

Comment (jakirkham) [inline recipes/basemap-data-hires/meta.yaml]: This is why I almost always use `git add -p`. That way I can go, "Woops! That shouldn't be there.", just one more time. Sometimes that is just enough times. :smile:

**Pull request merged**